scalafix-core/shared/src/main/scala/scalafix/internal/rule/RuleCtxImpl.scala

```diff
@@ -48,18 +55,28 @@ case class RuleCtxImpl(tree: Tree, config: ScalafixConfig) extends RuleCtx {
     logger.elem(values: _*)
   }
 
-  def printLintMessage(msg: LintMessage, owner: RuleName): Unit = {
+  def reportLintMessage(msg: LintMessage, owner: RuleName): Boolean = {
     val key = msg.category.key(owner)
-    if (config.lint.ignore.matches(key)) ()
```

Comment:
Can we maybe implement this filtering in `Patch.lintMessages` and this method reports everything?